### ISSUE TITLE
💅 `useVueValidVBind` incorrectly flags `v-bind="props"`

### ISSUE THREAD

**Issue by dyc3** [biomejs/biome#9347]
### Environment information

<details>

```bash

Paste output here

```

</details>

### Rule name

useVueValidVBind

### Playground link

https://biomejs.dev/playground/?lintRules=useVueValidVBind&code=PAB2AC0AZABpAGEAbABvAGcAIAB2AC0AbQBvAGQAZQBsAD0AIgBzAGgAbwB3ACIAIAB3AGkAZAB0AGgAPQAiADYAMAAwACIAPgAKAAkACQA8AHQAZQBtAHAAbABhAHQAZQAgAHYALQBzAGwAbwB0ADoAYQBjAHQAaQB2AGEAdABvAHIAPQAiAHsAIABwAHIAbwBwAHMAIAB9ACIAPgAKAAkACQAJADwAdgAtAGIAdABuACAAdgAtAGIAaQBuAGQAPQAiAHAAcgBvAHAAcwAiACAAcwB0AHkAbABlAD0AIgBtAGEAcgBnAGkAbgA6ACAAMAAgADIAMABwAHgAIgA%2BAAoACQAJAAkACQB7AHsAIAAkAHQAKAAiAGMAbABpAGUAbgB0AC0AcwBlAHQAdABpAG4AZwBzAC4AYQBjAHQAaQB2AGEAdABvAHIAIgApACAAfQB9AAoACQAJAAkAPAAvAHYALQBiAHQAbgA%2BAAoACQAJADwALwB0AGUAbQBwAGwAYQB0AGUAPgAKADwALwB2AC0AZABpAGEAbABvAGcAPgA%3D&language=vue

### Expected result

Apparently this is valid and shouldn't be flagged. Need to look into it a little more before confirming it.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
